recipes/node-url-to-whatwg-url/tests/url-parse/expected/file-10.js

```diff
@@ -0,0 +1,13 @@
+const url = require('node:url');
+
+const myURL = new URL('/path?query=string#hash');
```

Comment:
This throws an error

```
Uncaught TypeError: Failed to construct 'URL': Invalid URL
```

Reply:
@AugustinMauroy This codemod is very dangerous because it will cause apps to crash at runtime.

Reply:
@styfle if you can provide a better before/after example that can be used as test that will help us to fix this issue